fake_spectra/abstractsnapshot.py

```diff
@@ -164,10 +164,11 @@ def _get_all_files(self, num, base):
         Input:
             num - snapshot number
             base - simulation directory
-            file_num - file number in the snapshot"""
+            """
         fname = base
         snap=str(num).rjust(3,'0')
-        new_fname = os.path.join(base, "snapdir_"+snap)
+        #new_fname = os.path.join(base, "snapdir_"+snap)
+        new_fname = base
```

Comment:
This will break existing scripts. Could you instead fix loading snapshots called something else by adding an extra argument (call it snapprefix) to the constructor of AbstractSnapshot and propagating it down? HDF5Snapshot can have a default set as snapprefix = "snapdir_" and BigFileSnapshot can have as a default snapprefix="PART_"

Then _get_all_files can do:
new_fname = os.path.join(base, snapprefix+snap)